src/elm/Help/Commands.elm

```diff
@@ -721,6 +728,14 @@ resourceLoaded args =
         Pages.SourceRepositories ->
             args.sourceRepos.success
 
+        Pages.OrgRepositories _ ->
+            -- TODO: What should this be
+            args.sourceRepos.success
```

Comment:
Yup! @wass3r 

@JayCeeJr Because the Help Commands tray is generated dynamically, we have some records/helper functions that convert a WebData resource into a simpler loading/success struct called an `Arg`.

You can add the orgRepos as an `Arg` record field here:
https://github.com/go-vela/ui/blob/d620712a89ad938dcdacd77412ff3f1dde72d1d2/src/elm/Help/Commands.elm#L38-L54

Then using the `helpArg` function in Main.elm you can convert any `WebData a` value into a success/loading `Arg` that the Help Commands tray can use.
https://github.com/go-vela/ui/blob/d620712a89ad938dcdacd77412ff3f1dde72d1d2/src/elm/Main.elm#L2414-L2420
https://github.com/go-vela/ui/blob/d620712a89ad938dcdacd77412ff3f1dde72d1d2/src/elm/Main.elm#L2407-L2409